www/outsystems-secure-sqlite-init.js

```diff
@@ -1,7 +1,6 @@
 // Force dependency load
 var SQLiteCipher = require('cordova-sqlcipher-adapter.SQLitePlugin');
 var SecureStorage = require('cordova-plugin-secure-storage.SecureStorage');
-var Logger = require('com.outsystems.plugins.logger.OSLogger');
```

Comment:
Please rollback these changes, since this is the right way to invoke the Logger

Reply:
Good catch!! ✅